src/stories/DividerWayFinder.stories.js

```diff
@@ -1,28 +1,100 @@
-import DividerWayFinder from '@/lib-components/DividerWayFinder.vue'
+import { computed } from 'vue'
+import DividerWayFinder from '@/lib-components/DividerWayFinder'
 
 export default {
   title: 'DIVIDER / Way Finder',
   component: DividerWayFinder,
 }
 
-export const Default = {
-  args: {},
+// MOCK DATA
+const MockVisit = {
+  color: 'visit',
 }
 
-export const Visit = {
-  args: {
-    color: 'visit',
-  },
+const MockAbout = {
+  color: 'about',
 }
 
-export const About = {
-  args: {
-    color: 'about',
-  },
+const MockHelp = {
+  color: 'help',
 }
 
-export const Help = {
-  args: {
-    color: 'help',
-  },
+const MockFtva = {
+  color: 'ftva',
+}
+
+// STORIES
+export function Default() {
+  return {
+    components: { DividerWayFinder },
+    template: '<divider-way-finder />',
+  }
+}
+
+export function Visit() {
+  return {
+    data() {
+      return {
+        ...MockVisit,
+      }
+    },
+    provide() {
+      return {
+        theme: computed(() => 'visit'),
+      }
+    },
+    components: { DividerWayFinder },
+    template: `<divider-way-finder :color='color'/>`,
+  }
+}
+
+export function About() {
+  return {
+    data() {
+      return {
+        ...MockAbout,
+      }
+    },
+    provide() {
+      return {
+        theme: computed(() => 'about'),
+      }
+    },
+    components: { DividerWayFinder },
+    template: `<divider-way-finder :color='color'/>`,
+  }
+}
+
+export function Help() {
+  return {
+    data() {
+      return {
+        ...MockHelp,
+      }
+    },
+    provide() {
+      return {
+        theme: computed(() => 'help'),
+      }
+    },
+    components: { DividerWayFinder },
+    template: `<divider-way-finder :color='color'/>`,
+  }
+}
+
+export function FTVA() {
+  return {
+    data() {
+      return {
+        ...MockFtva,
+      }
+    },
+    provide() {
+      return {
+        theme: computed(() => 'ftva'),
+      }
+    },
+    components: { DividerWayFinder },
+    template: `<divider-way-finder :color='color'/>`,
```

Comment:
we do not want to have to write `<divider-way-finder :color="ftva">` every time to theme the component on the ftva site, so this should just be <divider-way-finder />